Add draft job (#65)
astropy/project: finance/jobs/2020_githubresponder.rst

> GitHub First/Last Responder

Reviewer: Fully agree

> GitHub First/Last Responder

Author: @kelle @eteq : The title came directly form the Coco. Please check with Coco if we can call it "operational support" instead.

> GitHub First/Last Responder

Reviewer 2: Operational Support Specialist sounds good to me.

> GitHub First/Last Responder

Author: "Operational support specialist" is the new term we came up with in the interim finance committee.

> GitHub First/Last Responder

Reviewer 2: GitHub Operational Support Specialist

> GitHub First/Last Responder

Reviewer 2: Software Development Operational Support Specialist? Astropy Operational Support Specialist?

> GitHub First/Last Responder

Reviewer 3: 👍 to Software, if you are taking votes.

> GitHub First/Last Responder

Reviewer 4: Astropy Development Support Specialist?
Astropy Developer Support Specialist?

GitHub First/Last Responder
===========================

The Astropy Project is looking to hire a Github first responder / last responder. The overarching goal of this position is to improve the first time contributor experience, reduce the technical barriers to contributions, reduce the number of open issues, and increase the number of merged pull requests. This is not intended to cover all issues and PRs opened in astropy, but rather to support and lighten the load on the rest of the volunteer community.

> An ideal candidate would be familiar with the astropy project ecosystem and capabilities, comfortable with the Github Development workflow, and have experience providing supportive feedback and technical advice in online environments.

Reviewer 3: Astropy's workflow is not just any dev workflow. For instance, someone familiar with Gitflow might not be familiar with the variant that we use, so better be a little more specific.
```suggestion
An ideal candidate would be familiar with the Astropy project ecosystem and capabilities, comfortable with its Github Development workflow, and has experience providing supportive feedback and technical advice in online environments.
```

> This role includes  monitoring  the astropy issue tracker, add labels to new issues, provide a friendly welcome message on issues and pull requests, and give starting points for how to resolve the issue or move the pull request forward. The “last responder” aspect of the role includes helping to close out issues and PRs, helping with rebasing, and troubleshooting failing tests.

Reviewer 3: Nitpick:
```suggestion
This role includes monitoring the astropy issue tracker, add labels to new issues, provide a friendly welcome message on issues and pull requests, and give starting pointers for how to resolve the issue or move the pull request forward. The “last responder” aspect of the role includes helping to close out issues and PRs, helping with rebasing, and troubleshooting failing tests.
```

We expect a commitment of about 8-10 hours/week for 12 months. The Astropy Project can pay inside or outside of the US. Compensation could be provided as a consulting fee or as a subaward. This is a remote-only position.

> Please send a short (one paragraph) email to finance@astropy.org with a description of your interest in the role, your qualifications, and any relevant experience.  If possible, please provide links to a few examples of issues/PRs where you have been involved as a fist and/or last responder.

Reviewer 3: ```suggestion
Please send a short (one paragraph) email to finance@astropy.org with a description of your interest in the role, your qualifications, and any relevant experience.  If possible, please provide links to a few examples of issues/PRs where you have been involved as a first and/or last responder.
```

> Please send a short (one paragraph) email to finance@astropy.org with a description of your interest in the role, your qualifications, and any relevant experience.  If possible, please provide links to a few examples of issues/PRs where you have been involved as a fist and/or last responder.

Reviewer 5: I'd also include a request for a CV

Reviewer 3: References would be nice too, if that is not too much to ask.

Author: The intention is that applying (and reviewing) is easy. This is intended (by CoCo decision) to be open to current astropy-maintainers only. We know those people and their experience. If noone steps forward, and we need to open this to the wider community, then we'll need to ask for more extensive documentation (CV, references).

Reviewer: oh, that's answers my biggest concern then (e.g. how would this ever work with a brand new person, as I don't feel this is the limiting bottleneck and not unavailable maintainer reviews), thanks @hamogu for the clarification.

The Astropy project is committed to fostering an inclusive community. The community of participants in open source Astronomy projects is made up of members from around the globe with a diverse set of skills, personalities, and experiences. It is through these differences that our community experiences success and continued growth. See https://www.astropy.org/code_of_conduct.html for details.

> The Astropy Project specfically encourages underrepresented parts of the community to apply.

Reviewer 3: ```suggestion
The Astropy Project specifically encourages underrepresented parts of the community to apply.
```

> All applications received less than four weeks after this PR is merged will receive full considerations, but you can apply until the position is filled.

Reviewer 3: Maybe put the exact date and time (with timezone) here right before merge. Otherwise, people need to constantly check the PR timestamp.

Author: That's the plan.

Context
=======

This is the first part time job ad / call for work proposals funded from a grant from the Moore foundation to the Astropy project (and administered by NumFOCUS). One purpose of this grant is to make life easier for all the volunteers who do the hard work of developing the Astropy Project by providing salary for some work that is very important for the project as a whole, but requires time commitments that are unreasonable to expect from volunteer contributors. This job is not intended to replace current volunteer efforts, but to help out with it and ensure that someone will do the work described below even if volunteers are not available.

> If you have suggestions on the process of job ads and spending grant money in general, please contact the interim finance committee at  finance@astropy.org or use the issue tracker of astropy/project.

Reviewer 3: ```suggestion
If you have suggestions on the process of job ads and spending grant money in general, please contact the interim finance committee at finance@astropy.org or use the issue tracker of astropy/project.
```

> If you have suggestions on the process of job ads and spending grant money in general, please contact the interim finance committee at  finance@astropy.org or use the issue tracker of astropy/project.

Reviewer 5: Include the link to the astropy/project

GitHub First/Last Responder
===========================

The Astropy Project is looking to hire a Github first responder / last
responder. The overarching goal of this position is to improve the
first time contributor experience, reduce the technical barriers to
contributions, reduce the number of open issues, and increase the
number of merged pull requests. This is not intended to cover all
issues and PRs opened in astropy, but rather to support and lighten
the load on the rest of the volunteer community.  An ideal candidate
would be familiar with the Astropy project ecosystem and capabilities,

> comfortable with its Github Development workflow, and has experience

Reviewer: not sure how to put this into the ad, but this role requires advanced `git` knowledge and comfort in working with multi fork environments. Resolving stalled PRs basically always requires rebases, squashes, cherry-picks, sorting out messed up histories, while keeping contributor credits.

providing supportive feedback and technical advice in online
environments.

> This role includes monitoring the astropy issue tracker, add labels to

Reviewer 4: “adding labels”. And below, “providing”, “giving”, etc.

new issues, provide a friendly welcome message on issues and pull
requests, and give starting pointers for how to resolve the issue or
move the pull request forward. The “last responder” aspect of the role
includes helping to close out issues and PRs, helping with rebasing,
and troubleshooting failing tests.

We expect a commitment of about 8-10 hours/week for 12 months. We aim
for an equitable pay with a total cost (incl. overhead, fringe etc.)

> <= 150 $/h to the project.

Author: We expect a commiment of up to 10 hours per week with a total cost compensation rate (which includes overhead and fringe) up to 150 $ per hour.

Compensation could be provided as a consulting fee or as a subaward
and since overheads and fringe benefits are handled differently in
these cases, the exact rate will be set to account for that.
The Astropy Project can pay inside or outside of the US. This is a
remote-only position.

Please send a short (one paragraph) email to finance@astropy.org with
a description of your interest in the role, your qualifications, and
any relevant experience.  If possible, please provide links to a few
examples of issues/PRs where you have been involved as a first and/or

> last responder.

Reviewer: 👍

The Astropy project is committed to fostering an inclusive
community. The community of participants in open source Astronomy
projects is made up of members from around the globe with a diverse
set of skills, personalities, and experiences. It is through these
differences that our community experiences success and continued
growth. See https://www.astropy.org/code_of_conduct.html for details.

The Astropy Project specifically encourages underrepresented parts of

> the community to apply.

Reviewer: 💯 - though it's funny, most of our diversity will be concentrated in this role, like the nursing of development...

Reviewer 3: Hah... Baby steps...
